content/wiki/raycasting/index.md

```diff
@@ -18,10 +18,19 @@ adapted to 3D, is *exactly* the thing one needs to travel along a line (or a ray
 
 {{ stub_notice() }}
 
-## Implementation
+## Ray Creation
+With raycasting or raytracing, we need to construct a ray for each pixel on the screen, represented by the camera. The following implementation is based on [this library](https://github.com/dps/rust-raytracer/tree/main/raytracer). It should be noted, that this is mostly for demonstration pourposes, and its probably not going to be that fast in compareison to other ray generation algorithms.
```

Comment:
Raytracing builds on top of raycasting, so this can just start with
> To raycast, [...]

There shouldn't be a comma after "noted".
> It should be noted [...]

"comparison"
> compareison